Workbooks/AKS/Deployments and HPAs AKS Engine/Deployments and HPAs AKS Engine.workbook

```diff
@@ -60,19 +68,18 @@
             "multiSelect": true,
             "quote": "'",
             "delimiter": ",",
-            "query": "InsightsMetrics\r\n| where Name contains \"kube_\"\r\n| extend Deploy = parse_json(Tags)\r\n| extend k8sNamespace = tostring(Deploy.k8sNamespace)\r\n| distinct k8sNamespace\r\n| project k8sNamespace",
-            "value": [
-              "value::all"
-            ],
+            "query": "InsightsMetrics\r\n| where Name contains \"kube_\"\r\n| extend Deploy = parse_json(Tags)\r\n| where Deploy[\"container.azm.ms/clusterId\"] =~ tostring('{clusterId}')\r\n| extend k8sNamespace = tostring(Deploy.k8sNamespace)\r\n| distinct k8sNamespace\r\n| project k8sNamespace",
```

Comment:
Please make a whereClusterIdClause parameter like we do in all the new workbooks, and add that to all the queries. This will also help transition this to an all-in-one workbook. I can sit with you for a few minutes tomorrow if you need some more context on how to do this. How urgent does this need to go in? I will potentially have the all-in-one workbook implementation details tested and ironed out by Friday. Then, you can just transition this workbook to an all-in-one workbook. It shoudl take no more than 30 minutes from start to finish. Otherwise, if this is urgent, we can push this through. I don't want to hold it up.